src/cli.ts

```diff
@@ -169,5 +176,111 @@ program
 				console.error('Invalid action. Use "getip" or "setip".');
 		}
 	});
+program
+	.command('setbrightness')
+	.description('Send http request for changing brightness.')
+	.argument('<Mode>', 'Set to either Enabled or Disabled.')
+	.argument('<Type>', 'Either absolute(A) or relative(R)')
+	.argument(
+		'<Value>',
+		'The brightness being set, if mode set to Absolute/A,0-100, if mode set to Relative, integer from -100 to 100.',
+	)
+	.option('--ip <ip>', 'The IP address of the Twinkly device')
+	.action(async (Mode, Type, Value, options) => {
```

Comment:
typically we take variable names as lower case.  Reserve upper case for class names, interfaces, that kinda thing.